apps/admin/src/app/(api)/api/v1/script_accepted/route.ts

```diff
@@ -22,12 +24,11 @@ export const POST = async (req: NextRequest) => {
             const sensorId = await getSensorIdFromSensorToken(data.accessToken);
 
             await updateNeedsScript(sensorId, false);
-
+            waitUntil(log("sensor/accepted-script", "info", "accepting-script", "api", data));
             return new NextResponse(ScriptAcceptedResponse.toBinary({ status: 200 }), { status: 200 });
         } catch (err) {
-            // eslint-disable-next-line no-console -- we need to log the error in the production logs
-            console.error(err);
             if ((err as unknown as Error).message === "token/expired") {
+                waitUntil(logError("sensor-script/token-expired", "accepting-script", "api", data, err));
```

Comment:
Keine Ahnung, ob das hier umsetzbar ist. Aber zumindestens aus dem ASP.NET Umfeld gibt es für Endpunkte die Möglichkeit, Middlewares hinzuzufügen. Da könnte man dann Prüfen, ob das Ergebnis des Endpunkts erfolgreich war und dementsprechend dann loggen. Das wäre aus meiner Sicht ein besser Variante, weil man dann wenig Anpassungsbedarf in Zukunft hat, wenn man mal was an den Endpunkten ändert, und das direkt bei allen Endpunkten anwendbar wäre.
Ich weiß aber wie gesagt nicht, ob das hier technisch machbar ist

Reply:
Ich will aber ja genauere Fehlermeldungen im Log haben, die ich hier erstellt habe - das ist doch eigentlich nicht schlecht oder?

Reply:
Mit Middleware wird schwierig, zumindest in NextJS, müssten wir eigene Wrapper bauen.
In anderen Frameworks aus der Typescript Welt (Hono, Fastify, Express) ist das einfach möglich